six/modules/c++/six.sidd/source/DerivedXMLParser200.cpp

```diff
@@ -1872,34 +1910,37 @@ XMLElem DerivedXMLParser200::convertDigitalElevationDataToXML(
     return dedElem;
 }
 
-void DerivedXMLParser200::parseGeographicTargetFromXML(
-    const XMLElem geographicElem,
-    GeographicAndTarget& geographicAndTarget) const
+void DerivedXMLParser200::parseGeoDataFromXML(
+    const XMLElem geoDataXML, GeoDataBase* geoData) const
 {
-    std::string model;
+    common().parseEarthModelType(getFirstAndOnly(geoDataXML, "EarthModel"),
+                        geoData->earthModel);
 
-    common().parseEarthModelType(getFirstAndOnly(geographicElem, "EarthModel"), geographicAndTarget.earthModel);
-    common().parseFootprint(getFirstAndOnly(geographicElem, "ImageCorners"), "ICP",
-        *geographicAndTarget.imageCorners);
+    common().parseFootprint(getFirstAndOnly(geoDataXML, "ImageCorners"), "ICP",
+                   geoData->imageCorners);
```

Comment:
so not to be that person, but what decided the amount of white space for second line? seems like -8 from the open paren on previous line?

Reply:
i should have saved my white space commentary for line 1928... unfortunately i am allowed only 1 white space comment per review...

Reply:
LOL.
In my defense 1928 isn't my code!
But also not in my defense, shouldn't have blindly copied it from SICD ...